rebench/executor.py

```diff
@@ -263,13 +263,15 @@ class Executor(object):
     def __init__(self, runs, do_builds, ui, include_faulty=False,
                  debug=False, scheduler=BatchScheduler, build_log=None,
                  artifact_review=False, use_nice=False, use_shielding=False,
-                 print_execution_plan=False, config_dir=None):
+                 print_execution_plan=False, config_dir=None,
+                 disable_denoise=False):
```

Comment:
Hm, could we possibly flip the semantics of this switch?
I find it often hard to reason about double negations, which we have later.
So, `use_denoise` would avoid the `not` later on, I think.